modules.d/01fips/fips.sh

```diff
@@ -17,6 +17,11 @@ fi
 mount_boot() {
     boot=$(getarg boot=)
 
+    if [ -z "$boot" ]; then
+        # try to retrive boot partition info from /sysroot/etc/fstab.
+        boot="$(sed -nE 's@^([^[:space:]]+)[[:space:]]+/boot[[:space:]]+.*$@\1@p' /sysroot/etc/fstab)"
```

Comment:
Wouldn't this be simpler?

```suggestion
        boot="$(grep -o '^\s*\S*\s*/boot\s*' /sysroot/etc/fstab)"
```

Also, shouldn't it be `${dracutsysrootdir}`? [[ref](https://github.com/dracutdevs/dracut/blob/master/docs/README.cross)]

@johannbg should I create a PRs fixing it across various modules?
https://github.com/dracutdevs/dracut/search?q=/sysroot